src/models/SCLedger.ts

```diff
@@ -0,0 +1,6 @@
+export default interface Ledger {
```

Comment:
btw, why do we need to store is like `[key]: value`?
Why don't we store everything like:
```TS
export default interface Ledger {
  original: string,
  derivative: string,
}
```
We can use it easily + will shortcut the `Object.keys Object.values` operations, but in this case we might lose easy access by `Ledger[originalAddress]`, so we need to think about it